bin/start.sh

```diff
@@ -17,7 +17,11 @@ if [ ! -d $MICA_HOME/conf ]
 	cp -r /usr/share/mica2/conf/* $MICA_HOME/conf
 	# So that application is accessible from outside of docker
 	sed s/address:\ localhost//g $MICA_HOME/conf/application.yml > /tmp/application.yml && \
+			echo "" >> /tmp/application.yml && \
+			echo "micaSearchPlugin.mica-search-es8: mica-search-es8" >> /tmp/application.yml && \
```

Comment:
Are you sure with this one?
It is already the default:
https://github.com/obiba/mica2/blob/master/mica-core/src/main/java/org/obiba/mica/micaConfig/service/PluginsService.java#L57-L59